c2corg_api/scripts/migration/documents/books.py

```diff
@@ -0,0 +1,108 @@
+from c2corg_api.models.book import Book, ArchiveBook, BOOK_TYPE
+from c2corg_api.models.document import DocumentLocale, ArchiveDocumentLocale, \
+    DOCUMENT_TYPE
+from c2corg_api.scripts.migration.documents.document import MigrateDocuments
+from c2corg_api.scripts.migration.documents.routes import MigrateRoutes
+
+
+class MigrateBooks(MigrateDocuments):
+
+    def get_name(self):
+        return 'books'
+
+    def get_model_document(self, locales):
+        return DocumentLocale if locales else Book
+
+    def get_model_archive_document(self, locales):
+        return ArchiveDocumentLocale if locales else ArchiveBook
+
+    def get_document_geometry(self, document_in, version):
```

Comment:
Yes and no... If I remove it then it crashes during migration -

Traceback (most recent call last):
  File "/home/dkocich/projects/c2corg/v6_api/c2corg_api/scripts/migration/migrate.py", line 93, in <module>
    main()
  File "/home/dkocich/projects/c2corg/v6_api/c2corg_api/scripts/migration/migrate.py", line 83, in main
    MigrateBooks(connection_source, session, batch_size).migrate()
  File "/home/dkocich/projects/c2corg/v6_api/c2corg_api/scripts/migration/documents/document.py", line 76, in migrate
    self._migrate(locales=False)
  File "/home/dkocich/projects/c2corg/v6_api/c2corg_api/scripts/migration/documents/document.py", line 146, in _migrate
    document_in, version)
  File "/home/dkocich/projects/c2corg/v6_api/c2corg_api/scripts/migration/documents/document.py", line 61, in get_document_geometry_archive
    doc['id'] = document_in.document_archive_id
TypeError: 'NoneType' object does not support item assignment